.github/workflows/react_tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: react-tests
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+    paths:
+    - client/
+    - **.js
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Dependencies
+      run: yarn
```

Comment:
We're using `npm`. Will using `yarn` in our actions cause problems?

Reply:
Honestly, I have no idea. It seems to work, but 🤷 . We should probably hold off on this until we actually have unit tests in js too